model/src/main/proto/interaction_object.proto

```diff
@@ -53,5 +54,30 @@ message Fraction {
 
 // Structure for a ListOfString object.
 message ListOfSetsOfHtmlStrings {
-  repeated StringList set_of_html_strings = 1;
+  StringList set_of_html_strings = 1;
+}
+
+// Structure for a ImageWithRegions object.
+message ImageWithRegions {
+  string image_path = 1;
+  repeated LabelRegions label_regions = 2;
+}
+
+message LabelRegions {
+  string label = 1;
+  Region region = 2;
+}
+
+message Region {
+  string region_type = 1;
+  NormalizedRectangle2D area = 2;
+}
+
+message NormalizedRectangle2D {
+  message Point2d {
+    float x = 1;
+    float y = 2;
+  }
+  Point2d lower_left = 1;
```

Comment:
Did you confirm that the lower left of the rectangular region is ``(0, 0)``? In many cases the origin is in the upper left and, if that applies here, the names of these properties would need to change.

Reply:
yeah it was supposed to be started from upper

Reply:
done